udata/core/spatial/commands.py

```diff
@@ -218,31 +146,33 @@ def migrate():
     Should only be run once with the new version of geozones w/ geohisto.
     '''
     counter = Counter(['zones', 'datasets'])
-    qs = GeoZone.objects.only('id', 'level', 'successors')
-    # Iter over datasets with zones
+    qs = GeoZone.objects.only('id', 'level')
+    # Fetch datasets with non-empty spatial zones
     for dataset in Dataset.objects(spatial__zones__gt=[]):
         counter['datasets'] += 1
         new_zones = []
         for current_zone in dataset.spatial.zones:
             counter['zones'] += 1
-            level, code, validity = geoids.parse(current_zone.id)
-            zone = qs(level=level, code=code).valid_at(validity).first()
+
+            level, code = geoids.parse(current_zone.id)
+            zone = qs(level=level, code=code).first() or qs(code=code).first()
+
             if not zone:
                 log.warning('No match for %s: skipped', current_zone.id)
```

Comment:
When testing today, I still have warnings on `No match for country:fr: skipped`, `country-subset:fr:dromcom`, `country-group:world: skipped`, etc.
Shouldn't this have been fixed?